lib/http/client.cpp

```diff
@@ -92,8 +184,10 @@ Client::Client(const std::string &server, uint16_t port)
         using boost::asio::ssl::context;
         p->ssl_ctx_.set_options(context::default_workarounds | context::no_sslv2 |
                                 context::no_sslv3 | context::no_tlsv1 | context::no_tlsv1_1);
-#ifdef WIN32
+#if WIN32
```

Comment:
Well, that works too, but ifdef is actually the correct way to do that imo?

Reply:
I'm not actually sure why I changed this to be honest.  I think I was throwing stuff at the wall to see what stuck for a while =D.  I'll change it back.

Reply:
Probably because there is no elifdef? But then you would use elif defined() ;p